packages/pwa-kit-dev/src/utils/script-utils.ts

```diff
@@ -290,18 +290,38 @@ export const createBundle = async ({
                 } = await getProjectPkg()
                 const extendsTemplate = 'node_modules/' + ccExtensibility.extends
 
+                const pwaKitDeps = [
+                    '@salesforce/pwa-kit-dev',
+                    '@salesforce/pwa-kit-runtime',
+                    '@salesforce/pwa-kit-react-sdk'
+                ]
```

Comment:
[Aside] I wonder what sort of insights we might get if we track _all_ `@salesforce` packages. Would that be useful at all?